product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Installing software
```

Comment:
Do we need PGE version of this?

Reply:
As in do we need to add one or as in do we need to include the PGE summary thats at the end of the document?

Reply:
Yeah I am basically debating whether the summaries are enough. I am slightly worried the shared preload libraries especially, and the different paths in the explanation text might be confusing. Not sure whether I over worry, but maybe at least paragraph that things are slightly different for PGE/PG and we provide summary at the end for those could be added at the beginning of "## Installing PGD for EDB Postgres Advanced Server" maybe?

Reply:
Restructured to be clearer and multiple version handling front loaded.